requirements/admin.txt

```diff
@@ -1,67 +1,68 @@
-apache-libcloud==1.0.0
+apache-libcloud==1.2.1
 # The `aws` command is required by the ``admin.installer``,
 # `admin.test.test_installer` and in the release process.
-awscli==1.10.44
-bitmath==1.3.0.2
-boto==2.41.0
+awscli==1.10.67
+bitmath==1.3.1.2
+boto==2.42.0
 characteristic==14.3.0
 deb-pkg-tools==1.36
-docker-py==1.8.1
+docker-py==1.10.3
```

Comment:
this probably brings some major changes. E.g. supporting `swarm-mode` which may be useful in testing things in the future with docker api instead of container api.

https://github.com/docker/docker-py/blob/master/docs/change_log.md#1103